csharp/ql/test/library-tests/comments/RegionDirectives.ql

```diff
@@ -0,0 +1,8 @@
+import csharp
+
+query predicate regionDirectives(RegionDirective d, string name, EndRegionDirective end) {
+  d.getName() = name and
+  d.getEnd() = end
+}
+
+query predicate endregions(EndRegionDirective d, RegionDirective start) { d.getStart() = start }
```

Comment:
I don't think we need this test as it is just a subset of the test above.

Reply:
Yep, it is just testing if `getStart()` is working, but I guess I could have just written `result.getEnd() = this` to make sure they are using the same relation.